lib/core/src/display/scene.rs

```diff
@@ -232,6 +239,10 @@ impl {
         }
     }
 
+    pub fn camera(&self) -> Camera2d {
+        self.camera.clone()
```

Comment:
use `clone_ref` to indicate that it is a cheap clone.

Reply:
Fixed